whatwg.org/irc

```diff
@@ -58,6 +58,10 @@
 
 <p>The simplest way to get started with IRC, if you are not familiar, is by signing up for a free <a href="https://www.irccloud.com/">IRCCloud</a> account. Once you've done that, you should be logged in to the Freenode server by default. All you'll have to do is join the #whatwg channel. If you tell your browser to use IRCCloud for irc:// links, then just <a href="ircs://irc.freenode.net/whatwg">clicking this link</a> should also take you there.</p>
 
+<p>Also, it is highly recommended that you register your nick on Freenode. For more information, read <a href="https://freenode.net/kb/answer/registration">this page</a>.</p>
+
+<p class="note">In view of ongoing spam messaging on Freenode, we highly suggest you set mode to <pre>+r</pre> in order to block unregistered users from sending you messages. For more information on user modes and how to set them, read <a href="https://freenode.net/kb/answer/usermodes">this page</a>.</p>
```

Comment:
This is probably good to mention as well, yes. Again, I'd suggest linking "you set mode to +r" instead to avoid an additional sentence.